Makefile

```diff
@@ -48,6 +48,7 @@ ifneq ($(CLEAN_BUILD),)
 endif
 
 GOLANGCI_LINT_VERSION = v1.61.0
+BASH_VERSION = v4.2.0
```

Comment:
@jpinsonneau - just to confirm, this has to be 4.2 or some other version? I just suggested from vague memory :)

Reply:
Associative array been introduced in bash 4.0 alpha but been polished in 4.1 & 4.2 so it's safer to require 4.2 and more I guess.
If you are able to ensure it works on `4.0.0` I can change the value here.

There are still some fixes in the next versions but I feel less important than the firsts. WDYT ?

https://github.com/bminor/bash/blob/master/CHANGES

Reply:
I am cool with recommending 4.2 then, thanks!